content/items/vunit.md

```diff
@@ -7,26 +7,30 @@ authors:
 links:
   gh: VUnit/vunit
   web: http://vunit.github.io
-date: "2014-11-25"
```

Comment:
I have a question here. I changed `date` by `active` (with subfields `from` and `to`), basically, because we defined it into *template.md*. I just saw that `date` is rendered (ej: https://hdl.github.io/awesome/items/pyvhdlparser/). I prefer `active` to specify a range instead of a specific `date`, because a lot of times (older projects) will be complicated to obtain. Let me know what do you think about that @umarcor, to revert here if needed.

Reply:
Your change is correct. `date` is previous to the change you did to the `template` for adding `from` and `to` fields. We should update the theme and update the items which don't have `from` yet.